doc/topics/cloud/qs.rst

```diff
@@ -0,0 +1,128 @@
+.. _salt-cloud-qs:
+
+=====================
+Salt Cloud Quickstart
+=====================
+
+Salt Cloud is built-in to Salt and is configured on and executed from your Salt
+Master. On some platforms, Salt Cloud is installed by the ``salt-cloud``
+package.
+
+This quickstart walks you through the basic steps of setting up a cloud host
+and defining some virtual machines to create.
+
+Define a Provider
+-----------------
+The first step is to add the credentials for your cloud host. Credentials
+and other settings provided by the cloud host are stored in provider configuration files.
+Provider configurations contain the details needed to connect to a cloud host such as EC2, GCE, Rackspace, etc.,
+and any global options that you want set on your cloud minions (such as the location of your Salt Master).
+
+On your Salt Master, browse to ``/etc/salt/cloud.providers.d/`` and create a file called ``<provider>.provider.conf``,
+replacing ``<provider>`` with ``ec2``, ``softlayer``, and so on. The name helps you identify the contents, and is not
+important as long as the file ends in ``.conf``.
+
+Next, browse to the :ref:`Provider specifics <cloud-provider-specifics>` and add any required settings for your
+cloud host to this file. Here is an example for Amazon EC2:
+
+.. code-block:: yaml
+
+    my-ec2:
+      driver: ec2
+      # Set the EC2 access credentials (see below)
+      #
+      id: 'HJGRYCILJLKJYG'
+      key: 'kdjgfsgm;woormgl/aserigjksjdhasdfgn'
+      # Make sure this key is owned by root with permissions 0400.
+      #
+      private_key: /etc/salt/my_test_key.pem
+      keyname: my_test_key
+      securitygroup: default
+      # Optional: Set up the location of the Salt Master
+      #
+      minion:
+        master: saltmaster.example.com
+
+The required configuration varies between cloud hosts so make sure you read the provider specifics.
+
+List Cloud Provider Options
+---------------------------
+You can now query the cloud provider you configured for available locations,
+images, and sizes. This information is used when you set up VM profiles.
+
+.. code-block:: bash
+
+    salt-cloud --list-locations <provider_name>  # my-ec2 in the previous example
+    salt-cloud --list-images <provider_name>
+    salt-cloud --list-sizes <provider_name>
+
+Replace ``<provider_name>`` with the name of the provider configuration you defined.
+
+Create VM Profiles
+------------------
+On your Salt Master, browse to ``/etc/salt/cloud.profiles.d/`` and create a file called ``<provider>.profiles.conf``,
```

Comment:
Remove the `profiles` part from <provider>.profiles.conf. Should just be <profile-name>.conf or something like that.

Notice that the `<provider>.profiles.conf` also says "provider" instead of "profile". Make sure it's `<profile>.conf` and not provider.